code/__HELPERS/_logging.dm

```diff
@@ -175,6 +175,10 @@ GLOBAL_LIST_INIT(testing_global_profiler, list("_PROFILE_NAME" = "Global"))
 	if (CONFIG_GET(flag/log_prayer))
 		WRITE_LOG(GLOB.world_game_log, "PRAY: [text]")
 
+/proc/log_craft(text)
+	if (CONFIG_GET(flag/log_craft))
+		WRITE_LOG(GLOB.world_game_log, "GAME: [text]")
```

Comment:
Okay wait, why do we need this proc at all instead of just using `log_game`?

Reply:
Hell, `log_message` on its own *already* does log_game.

I don't think you need this at all, which means you also don't need the config and the bitflag (we're getting really close to the limit on that), and just normal `log_message` should be fine.

Reply:
You also never use this proc to begin with, so the config never mattered.

Reply:
@Mothblocks If I read the code correctly, log_message calls log_craft unless I had a moment. I'm fine with just logging it through game though.

Reply:
You had a moment

Reply:
😔